topics/setup.md

```diff
@@ -6,13 +6,13 @@ Before you begin [creating your first application](create-first-app.md) to work
 1. If you are going to work with shared code or Android-specific code, you can work on any computer with an operating 
    system supported by [Android Studio](https://developer.android.com/studio).  
    If you also want to write iOS-specific code and run an iOS application on a simulated or real device, use a Mac with a 
-   macOS.  
+   macOS. These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.
 2. Install [Android Studio](https://developer.android.com/studio) – version 4.1 or higher.   
     You will use Android Studio for creating your multiplatform applications and running them on simulated or hardware devices.
-3. If you need to write iOS-specific code and run an iOS application, install [Xcode](https://apps.apple.com/us/app/xcode/id497799835) 
+3. If you need to write iOS-specific code and run an iOS application, install [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
     –  version 11.3 or higher.                                                                                                                                                                                                                                                                                                                          
     Most of the time, Xcode will work in the background. You will use it to add Swift or Objective-C code to your iOS application.
-4. Update the Kotlin plugin to version 1.4.20 or higher.  
+4. If you have used your current installation of Android Studio before, update the Kotlin plugin to version 1.4.20 or higher.  
```

Comment:
Because if not, then you don't need to do anything. If you have just installed AS – everything is already up to date.

Reply:
Maybe rephrase. 
> Make sure that you have a [compatible Kotlin plugin] installed.

I would remove a specific Kotlin version here.